### DISCUSSION TITLE
How the Vue official documentation website manages SSR?

### DISCUSSION THREAD

**Discussion by kiranparajuli589** [vuejs/docs#2407] [Q&A] [answered]
I have cloned the vue official documentation repository locally, run it locally and checked the `Page Source`. But there is nothing related to the `site-meta`. Even the `title` tag is empty. I think every thing is managed using JS.

Now If i go to `https://vuejs.org/`, same webpage has all the beautiful `meta` set. How is this managed?

Why I am interested?
I have recently built a whole website using Vue3+Vuetify3 and now I realized SEO is completely missing. Setting the title or meta information using JS just does not work for SEO. Is there any method to handle this case without completely rewriting the code over Nuxt or other SSR tech.? Thank You.

**Comment by brc-dd** [comment, marked as answer]
It uses https://vitepress.dev/. The page source is empty in your case because you're checking it in dev mode. Run build and preview to get the HTML with content and head tags.

For your use case, its probably SSG what you need. Try using this: https://github.com/antfu/vite-ssg
